src/templates/index.pug

```diff
@@ -1,21 +1,32 @@
 include _common.pug
-+head(title="A Tour of C++")
++head(title="C++ Tour")
 
 mixin sidebar_link(id, title)
     a.sidebar__item(id=id href="#"+id)=title
 
 body.layout
     // Page header
     nav.layout__header#header
-        a.layout__header__menu="A Tour of C++"
-        span="Lesson name"
+        a.layout__header__menu(class=['headline'])="C++ Tour"
```

Comment:
In pug you can chain multiple classes just through like `a.layout__header__menu.headline`